src/reactviews/pages/QueryResult/table/utils.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export interface ISlickRange {
```

Comment:
is this logic ported from ADS or new for mssql?

Reply:
ported from ADS